lib/sablon/template.rb

```diff
@@ -32,6 +33,10 @@ def render(context, properties = {})
             out.write(process(Processor::Document, content, env))
           elsif entry_name == 'word/numbering.xml'
             out.write(process(Processor::Numbering, content))
+          elsif entry_name == 'word/_rels/document.xml.rels'
+            out.write(process(Processor::Image, content, properties))
```

Comment:
However, feel free to make changes! I'll be working on this and let you know soon